core/src/main/scala/kafka/coordinator/group/MemberMetadata.scala

```diff
@@ -29,8 +29,8 @@ case class MemberSummary(memberId: String,
                          metadata: Array[Byte],
                          assignment: Array[Byte])
 
-private object MemberMetadata {
-  def plainProtocolSet(supportedProtocols: List[(String, Array[Byte])]) = supportedProtocols.map(_._1).toSet
+object MemberMetadata {
+  private[group] def plainProtocolSet(supportedProtocols: List[(String, Array[Byte])]) = supportedProtocols.map(_._1).toSet
```

Comment:
I saw this compiler warning too and it seems like a compiler bug. Could you not share the object modifier to be `private[group]` instead? Either way, worth a comment since it's unnatural.